docs/quick-start/lua-howto.md

```diff
@@ -0,0 +1,95 @@
+---
+template: main.html
+---
+
+![Setup-Banner](https://github.com/ExpressLRS/ExpressLRS-Hardware/raw/master/img/quick-start.png)
+
+Now that you have flashed your TX it is time to learn how to use the ELRS lua script!
+
+*If you flashed via WiFi you need a different version of the lua than you already installed.*
```

Comment:
it might be better to say that "If you were previously on version prior to V2, you need a different version of the LUA than what you already installed/copied"